VmAgent.Core/Dependencies/FileWriteWrapper.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Gaming.VmAgent.Core.Dependencies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using ApplicationInsights;
+    using ApplicationInsights.DataContracts;
+    using global::VmAgent.Core.Dependencies.Interfaces.Exceptions;
+    using Microsoft.Azure.Gaming.VmAgent.Core.Dependencies.Interfaces;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public class FileWriteWrapper : IFileWriteWrapper
+    {
+        private string _logFilePath;
+        private TextWriter _textWriter;
+
+        public FileWriteWrapper()
+        {
+        }
+
+        public void CreateFile(string logFilePath)
+        {
+            _logFilePath = logFilePath;
+            _textWriter = TextWriter.Synchronized(new StreamWriter(File.OpenWrite(logFilePath)));
+        }
+
+        public bool Close()
+        {
+            if (_textWriter != null)
+            {
+                _textWriter.Close();
+                _textWriter = null;
+
+                return true;
+            }
+            return false;
+        }
+
+        public void Write(string message, string streamType = "stdout")
+        {
+            if (_textWriter != null)
+            {
+                string currentDate = DateTime.Now.ToString("yyyy-MM-dd'T'HH:mm:ss.fffK");
+
+                _textWriter.WriteLine($"{{\"log\":\"{message}\", \"stream\":{streamType}, \"time\":{currentDate}}}");
```

Comment:
I think this will output a log that looks like this:
`{{"log":"Some message", "stream":stdout, "time":2022-09-29T00:00:00.000}}`
1) Why is it surrounded with `{{` instead of just `{`?
2) The `streamType` and `time` values both need to be surrounded by `"` to be proper json

Reply:
1. {{ will be shown as { in txt and I did this to match the log file format with Docker. In PF_console.txt for Docker, each line has { } in it.
Ex: {"log":"Running Startup Script\n","stream":"stdout","time":"2022-06-15T21:35:47.5767715Z"}
2. gotcha

Reply:
Also I think you should call `JsonConvert.ToString(message)` to escape `message`--if  `message` contains characters like `"` or `{` it'll mess up the log line (I'm assuming we want the log line to be valid json--i don't think that's actually a requirement but it would probably be nice for developers)